### ISSUE TITLE
Unable to send config file

### ISSUE THREAD

**Issue by JayJay2105** [tasmota/tasmotizer#30]
Hi,
Flashed a Sonoff T2uk3ch using tasmotizer, that all works fine, powered off, disconnected gpio from gnd, powered on, try to send config of t1 3ch as per other users, reports sent ok, power off and on, not connecting to wifi, no ip address showing up in router, check in with termite, ipadress1 - reports ipUnSet. Wifi light ono Sonoff just flashes.

**Comment by jziolkowski** [comment]
Yes, I've noticed it recently as well. Check the fix in the development version (it will require manual installation of the required packages)

**Comment by silentcossack** [comment]
I tried the latest git develop version and I still can't send config. Flashing worked fine, but whenever i try to send config i get this "bool QSerialPort::flush(): device not open". I'm on Mac, using the python3 version.

**Comment by jziolkowski** [comment]
Please test again using release 1.2, and develop-1.3
